chatmd/importers/chatgpt.py

```
@@ -0,0 +1,387 @@
+    if not role:
+        return "assistant"
+    if role in {"user", "assistant", "tool", "system"}:
+        if role == "assistant":
+            return "model"
+        return role
+    if role == "function":
+        return "tool"
+    return "assistant"
+
+
+def _extract_text(role: Optional[str], content: Dict) -> str:
```

Ruff (0.15.9)

[warning] Unused function argument: `role`

(ARG001)